### PR TITLE
new icon: carbon (original)

### LINKED ISSUE #1895: [ICON REQUEST]: <Crabon>
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Carbon is an experimental programming language designed for interoperability with C++.

### Links and sources

https://github.com/carbon-language/carbon-lang

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by weh**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes**
closes #1895 

## Link to prove your SVG is correct and up-to-date.

https://github.com/carbon-language/carbon-lang

### CODE REVIEW

**Review by canaleal — COMMENTED** [review]
LGTM.

File: icons/carbon/carbon-original.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M53.316 127.094c-13.828-2.641-24.859-8.398-34.437-17.973C-1.914 88.328-5.875 56.875 9.074 31.309 17.551 16.813 31.715 6.371 49.047 1.836c8.141-2.129 21.766-2.129 29.906 0 14.188 3.711 25.898 11.066 34.387 21.586 6.461 8.02 10.141 15.367 12.824 25.625 2.129 8.141 2.129 21.766 0 29.906-6.277 24.004-23.273 41.016-47.125 47.18-6.629 1.715-19.293 2.188-25.723.961zm22.191-24.953c4.637-1.891 8.102-4.883 11.227-9.691 2.133-3.285 4.145-8.914 4.145-11.605 0-1.41-.219-1.484-4.492-1.484-4.996 0-4.973-.02-6.383 5.574-.922 3.66-4.203 7.77-7.684 9.625-2.488 1.328-3.582 1.508-8.047 1.328-4.32-.172-5.594-.488-7.746-1.91-3.113-2.062-6.289-6.687-7.605-11.066-2.383-7.949-2.66-26.223-.539-35.535 2.184-9.574 8.395-15.359 16.504-15.371 7.852-.012 13.488 4.969 15.754 13.918l.688 2.719h9.727l-.41-2.996c-.227-1.648-1.316-4.961-2.422-7.355-6.055-13.141-21.758-18.172-35.551-11.391-6.379 3.137-11.187 9.598-14.059 18.895-1.73 5.609-1.73 30.805 0 36.414 2.621 8.496 7.402 15.402 12.887 18.613 6.652 3.898 16.371 4.434 24.008 1.32zm0 0" fill="rgb(0%,0%,0%)"/></svg>
```
Comment (Snailedlt) [inline icons/carbon/carbon-original.svg:1]: Changed the fill to `#000000` 
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M53.316 127.094c-13.828-2.641-24.859-8.398-34.437-17.973C-1.914 88.328-5.875 56.875 9.074 31.309 17.551 16.813 31.715 6.371 49.047 1.836c8.141-2.129 21.766-2.129 29.906 0 14.188 3.711 25.898 11.066 34.387 21.586 6.461 8.02 10.141 15.367 12.824 25.625 2.129 8.141 2.129 21.766 0 29.906-6.277 24.004-23.273 41.016-47.125 47.18-6.629 1.715-19.293 2.188-25.723.961zm22.191-24.953c4.637-1.891 8.102-4.883 11.227-9.691 2.133-3.285 4.145-8.914 4.145-11.605 0-1.41-.219-1.484-4.492-1.484-4.996 0-4.973-.02-6.383 5.574-.922 3.66-4.203 7.77-7.684 9.625-2.488 1.328-3.582 1.508-8.047 1.328-4.32-.172-5.594-.488-7.746-1.91-3.113-2.062-6.289-6.687-7.605-11.066-2.383-7.949-2.66-26.223-.539-35.535 2.184-9.574 8.395-15.359 16.504-15.371 7.852-.012 13.488 4.969 15.754 13.918l.688 2.719h9.727l-.41-2.996c-.227-1.648-1.316-4.961-2.422-7.355-6.055-13.141-21.758-18.172-35.551-11.391-6.379 3.137-11.187 9.598-14.059 18.895-1.73 5.609-1.73 30.805 0 36.414 2.621 8.496 7.402 15.402 12.887 18.613 6.652 3.898 16.371 4.434 24.008 1.32zm0 0" fill="#000000"/></svg>
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Thank you for another great contribution! 💯 🙇
Again, I just suggested a small change for the icon, once that's added it's all good!

**Review by Snailedlt — APPROVED** [review]
Thank you for another great PR @weh! 💯 
Looking forward to the next one 🚀

**Review by canaleal — APPROVED** [review]
Devicon.json is great, and the SVG is optimized/ uses hex fill. Nice nice.

**Pull request merged**